### ISSUE TITLE
Docker container missing the bc package

### ISSUE THREAD

**Issue by uberjay** [github/backup-utils#1061]
Adding the progress-indicator feature (along with its associated `track-progress` helper script) broke backing up with the supplied Dockerfile. This script uses the `bc` tool, but it is not installed as a runtime dependency.

In other words:

```diff
diff --git a/Dockerfile b/Dockerfile
index 66773972..5cbc7a5c 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -50,6 +50,7 @@ FROM ubuntu:focal AS runtime

 # Install runtime dependencies -  bash, git, OpenSSH 5.6 or newer, and jq v1.5 or newer.
 RUN apt-get update && apt-get install --no-install-recommends -y \
+    bc \
     bash \
     git \
     openssh-client \
```

Otherwise:
```
2023-07-13T13:50:38Z  INFO  Starting backup of xxx with backup-utils v3.9.0 in snapshot yyy
Checking host for sufficient space for a backup...
We recommend allocating at least 5x the amount of storage allocated to the primary GitHub appliance for historical snapshots and growth over time.
Available space: ZZZ MB
Min Disk required for this backup is at least AAA MB

### Data Transfer Sizes
[snip]

Connect xxx.yyy.zzz:122 OK (v3.9.0)
Running on: Ubuntu 20.04.5 LTS
CPUs: N
Memory total/used/free+share/buff/cache: ZZZ
/backup-utils/share/github-backup-utils/track-progress: line 10: bc: command not found
2023-07-13T13:50:50Z  WARN  Warning: storing backup-utils version remotely failed.
2023-07-13T13:50:50Z  INFO  Backing up GitHub settings ...
/backup-utils/share/github-backup-utils/track-progress: line 10: bc: command not found
2023-07-13T13:50:54Z  INFO  Backing up SSH authorized keys ...
/backup-utils/share/github-backup-utils/track-progress: line 10: bc: command not found
```

(EDIT: the package is also missing from the Backup Host Requirements documentation)

**Comment by chuckp22** [comment]
@uberjay Thanks for bringing this up, the `bc` Dockerfile  and requirements doc updates were made in the last couple of weeks and should be available in the next patch release.

**Comment by Chocrates** [comment]
@chuckp22 what docker image version did we fix the `bc` issue?  Customer is running `3.9.1` docker image and missing the `bc` package, even though it looks like it was released after the fix.

**Comment by chuckp22** [comment]
@Chocrates Looks like it made it in [v3.9.2](https://github.com/github/backup-utils/blob/v3.9.2/Dockerfile#L29)
